stories/link/link.stories.js

```diff
@@ -34,7 +34,7 @@ export const primary = () => `
 <br><br>
 <a class="fd-link" aria-disabled="true">Disabled link</a>
 <br><br>
-<div style="background-color:#314a5e;padding:10px">
+<div class="fd-shellbar">
```

Comment:
you can just use the variable in the example, instead of bringing the shellbar in the link

Reply:
@droshev if we use the variable in the documentation in the style, can we use the same variable in NGX Documentaiton too?

Reply:
You need to use var(--sapShellColor)

Reply:
> @droshev if we use the variable in the documentation in the style, can we use the same variable in NGX Documentaiton too?

yes, you can

Reply:
<img width="1297" alt="Screen Shot 2021-08-30 at 11 49 57 AM" src="https://user-images.githubusercontent.com/39598672/131367255-4afc8b65-eb03-4544-98ae-4129f293065c.png">

Reply:
@InnaAtanasova but issue is not on the background, its on the focus by default for the link outline-color: var(--sapContent_FocusColor, #000);outline-style: var(--sapContent_FocusStyle, dotted); but the focus has to be differentiate with the light clour like hows its done for the button.